package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "superformula-cloud-backend-test",
+  "version": "1.0.0",
+  "description": "Serverless GrapqhQL for simple CRUD operations",
+  "author": "Mario Ruiz Diaz (https://www.linkedin.com/in/marioruizdiaz/)",
+  "license": "MIT",
+  "main": "handler.js",
+  "engines": {
+    "node": ">=14.0.0",
+    "npm": ">=6.0.0"
+  },
+  "scripts": {
+    "deploy": "serverless deploy",
+    "destroy": "serverless remove",
+    "local": "npm run docker:local-db && serverless offline start --apiKey super-formula-api-key-token",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "docker:local-db": "cd ./dynamodb && docker-compose up -d && cd ..",
+    "test:unit": "npx jest --coverage -t 'Unit'",
+    "test:integration": "npx jest --coverage -t 'Integration' ",
+    "test:e2e": "npx jest -t 'e2e'"
+  },
+  "dependencies": {
+    "@mapbox/mapbox-sdk": "^0.12.1",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@types/cwd": "^0.10.0",
+    "@types/uuid": "^8.3.0",
+    "apollo-datasource-rest": "^0.13.0",
+    "apollo-link-http": "^1.5.17",
+    "apollo-server-lambda": "^2.16.1",
+    "apollo-server-testing": "^2.24.0",
+    "cwd": "^0.10.0",
+    "graphql": "^15.3.0",
+    "graphql-request": "^3.4.0",
+    "node-fetch": "^2.6.1",
+    "source-map-support": "^0.5.10",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/aws-lambda": "^8.10.17",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^10.14.22",
+    "eslint": "^6.6.0",
+    "jest": "^26.6.3",
+    "serverless": "^1.79.0",
+    "serverless-dotenv-plugin": "^3.0.0",
+    "serverless-dynamodb-local": "^0.2.18",
+    "serverless-offline": "^6.5.0",
+    "serverless-plugin-typescript": "^1.1.9",
+    "ts-jest": "^26.5.6",
+    "ts-loader": "^5.3.3",
+    "tslint": "^5.20.0",
+    "tslint-config-airbnb": "^5.11.2",
+    "typescript": "^3.2.4"
+  },
+  "jest": {
```

Comment:
Are there any benefits to abstracting this into its own config file?

Reply:
The benefits are related to the have  less cluttered packages.json and having spacific setting in its related specific files.